### PR TITLE
COMMERCE-7198

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-7198

### CODE REVIEW

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase [hunk 1]
```diff
@@ -1139,6 +1139,59 @@ definition {
...
+			Click(
+				key_searchBarValue = "Search Order",
+				locator1 = "CommerceAccelerators#QUICK_ADD_TO_CART_EMPTY_SEARCH_BAR");
+
+			SelectFrameTop();
+
+			Type(
+				key_searchBarValue = "Search Order",
+				locator1 = "CommerceAccelerators#QUICK_ADD_TO_CART_EMPTY_SEARCH_BAR",
+				value1 = ${firstOrderId});
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase:1181]: A new macro can be created and used instead

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase [hunk 2]
```diff
@@ -1139,6 +1139,59 @@ definition {
 			value1 = "Showing 1 to 2 of 2");
 	}
 
+	@description = "COMMERCE-7198. I want to be able to search for one pending order using the search bar on the Account Selector"
+	@priority = 4
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOrders.testcase:1143]: As the test will be included in the relevant, please change the priority from 4 to 5

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@Brunno0liveira please follow our commit naming standards for your commits. Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by brianikim** [comment]
@Brunno0liveira , can you update this PR? I changed the commit message on da31f48.

Rest of your change should be based on the following:

See: https://github.com/liferay/liferay-portal/compare/master...brianikim:liferay-portal:COMMERCE-7198

**Review by lorenzolapescara — APPROVED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:reevaluate:1317401_7211

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by victoroscuro** [comment]
ci:forward

**Pull request closed without merging**